Remotion/Web/Development.WebTesting.UnitTests/App.config

```diff
@@ -0,0 +1,16 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <configSections>
+    <section name="remotion.webTesting"
+        type="Remotion.Web.Development.WebTesting.Configuration.WebTestConfigurationSection,Remotion.Web.Development.WebTesting" />
+  </configSections>
+  <remotion.webTesting xmlns="http://www.re-motion.org/WebTesting/Configuration/1.0"
+      browser="Chrome"
+      commandTimeout="00:13:37"

```

Comment:
Shouldn't there be a change to the schema file?

There's also examples in re-motion on how to test config sections.

Reply:
A schema file change was made [already](https://github.com/re-motion/Trunk-SVN2Git/pull/6/files#diff-27f09b194592c4d16a6b918b983a2025).

The issue [RM-7241 Add WebTestConfigurationSection tests](https://re-motion.atlassian.net/browse/RM-7241) was added, and will be merged before this PR.